libs/utils/src/lib/models/operationDiscoveryResults.ts

```diff
@@ -30,6 +30,7 @@ export interface SomeKindOfAzureOperationDiscovery {
   capabilities?: string[];
   isNotification?: boolean;
   api: OperationApi;
+  displayApi?: OperationApi;
```

Comment:
This can also be just an id, since the rest isn't used so far, but could be in the future. Would it better to just have a "displayApiId" ?